api/addon/register.go

```diff
@@ -0,0 +1 @@
+package addon
```

Comment:
Is this an empty file?

Reply:
Yes, currently it is an empty file marking this addon package.

Reply:
dcf19b9f2985404bafe65bcd4e7d3b20a0291713